adapters/adform/adform_test.go

```diff
@@ -519,11 +522,22 @@ func getUserExt() []byte {
 }
 
 func formatAdUnitJson(tag aTagInfo) string {
-	return fmt.Sprintf("{ \"mid\": %d%s%s%s}",
+	return fmt.Sprintf("{ \"mid\": %d%s%s%s%s%s%s}",
 		tag.mid,
```

Comment:
Are we sure we don't want to use `%d` for `tag.minp`? We could get a `%!s(int=XX) ` this way.

Reply:
Can't get how this could happen? https://github.com/prebid/prebid-server/blob/1c2e93c2c52a89e25fe33b2e2c02dc71a850126c/adapters/adform/adform_test.go#L537